docs/sources/alerting/manage-notifications/alertmanager.md

```diff
@@ -37,6 +37,8 @@ Here are two examples of when you may want to configure your own external alertm
 
 Alertmanagers are visible from the drop-down menu on the Alerting Contact Points, Notification Policies, and Silences pages.
 
+If you are provisioning your datasource, set the flag `handleGrafanaManagedAlerts` in the `jsonData` field to `true` to send Grafana-managed alerts to this Alertmanager.
```

Comment:
```suggestion
If you are provisioning your data source, set the flag `handleGrafanaManagedAlerts` in the `jsonData` field to `true` to send Grafana-managed alerts to this Alertmanager.
```

Reply:
I think this info belongs here more than on this page? of course we can add it to both for more visibility; but this topic is the concept -- this topic is how to add an external alertmanager: https://grafana.com/docs/grafana/latest/alerting/set-up/configure-alertmanager/
wdyt?